src/packages/console/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@kibalabs/core": "^0.5.3",
-    "@kibalabs/core-react": "^0.6.0",
+    "@kibalabs/core-react": "^0.6.1-next.0",
```

Comment:
if core-react is used in other packages please update there aswel

you will need to run npm bootstrap to make sure all are up to date

Reply:
done